requirements-tests.txt

```diff
@@ -2,3 +2,5 @@ invoke==0.11.1
 coverage==3.7.1
 flake8==2.2.5
 six==1.10.0
+
+-r requirements.txt
```

Comment:
This is breaking the tests because django gets specified twice. I we should just not include this.